charts/kyverno/values.yaml

```diff
@@ -357,7 +368,7 @@ serviceMonitor:
   additionalLabels:
     # key: value
   # -- Override namespace (default is the same as kyverno)
-  namespace:
+  namespace: kyverno
```

Comment:
This change made it into the README.md also, I think this is unrelated to this pull request.

Reply:
I didn't see it properly as it was added on local e2e tests 😨 nice catch